engine/release-notes.md

```diff
@@ -625,34 +632,10 @@ Ubuntu 14.04 "Trusty Tahr" [docker-ce-packaging#255](https://github.com/docker/d
 * Now using original process spec for execs. [moby/moby#38871](https://github.com/moby/moby/pull/38871)
 * Fixed [CVE-2018-15664](https://nvd.nist.gov/vuln/detail/CVE-2018-15664) symlink-exchange attack
 with directory traversal. [moby/moby#39357](https://github.com/moby/moby/pull/39357)
-=======
-## 18.03.1-ee-10
-
-2019-07-17
-
-### Runtime
-
-* Masked the secrets updated to the log files when running Docker Engine in debug mode. [CVE-2019-13509](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-13509): If a Docker engine is running in debug mode, and `docker stack deploy` is used to redeploy a stack which includes non-external secrets, the logs will contain the secret.
-
-## 18.03.1-ee-9
-
-2019-06-27
```

Comment:
Did you mean to delete the 2019-06-27 notes?

Reply:
@adrian-plata it was there twice, take a look with more context and you'll see